1-js/02-first-steps/16-arrow-functions-basics/article.md

```diff
@@ -33,79 +34,80 @@ let sum = function(a, b) {
 alert( sum(1, 2) ); // 3
 ```
 
-As you can, see `(a, b) => a + b` means a function that accepts two arguments named `a` and `b`. Upon the execution, it evaluates the expression `a + b` and returns the result.
+এখানে আপনি যেমনটি দেখতে পাচ্ছেন, `(a, b) => a + b` ফাংশনটি দুইটা আর্গুমেন্ট নিচ্ছে যথাক্রমে `a` ও `b` এবং সম্পাদনের সময় এটি `a + b` এক্সপ্রেশনটির মান নির্ণয় করছে এবং তার রেজাল্টটি রিটার্ন করছে। 
 
-- If we have only one argument, then parentheses around parameters can be omitted, making that even shorter.
+- যদি আমাদের কেবল একটি মাত্র আর্গুমেন্ট থাকে তাহলে প্যারামিটারগুলোর দুই পাশে যে প্যারেন্থেসিস বা প্রথম বন্ধনী থাকে সেটি না দিলেও চলে, যেটা কোডটাকে আরও সংক্ষিপ্ত করে নিয়ে আসে। 
 
-    For example:
+    উদাহরণ স্বরূপঃ
 
     ```js run
     *!*
     let double = n => n * 2;
-    // roughly the same as: let double = function(n) { return n * 2 }
+    // এটা বলেতে গেলে let double = function(n) { return n * 2 } অন্যরূপ।  
```

Comment:
এটা বলতে গেলে let double = function(n) { return n * 2 } এর অন্যরূপ

Reply:
Thank you. I will change it.